tests/fail-dep/libc/env-set_var-data-race.rs

```diff
@@ -1,5 +1,6 @@
 //@compile-flags: -Zmiri-disable-isolation -Zmiri-preemption-rate=0
-//@ignore-target-windows: No libc env support on Windows
+// No libc env support on Windows
+//@ignore-target: windows
```

Comment:
When grepping the repo it is quite nice to have the explanation for an `ignore` on the same line as the `ignore`... I wonder if there is any way that can be preserved? Like
```
//@ignore-target: windows # No libs env support on Windows
```
Not sure what looks better: `#` or `//` as separator here.

Reply:
I made that a feature request, it doesn't have to block this PR: https://github.com/oli-obk/ui_test/issues/277. (Though undoing all the multi-lining here if/when that feature ever gets implemented will be quite painful...)

Reply:
I gave this some thought, and doing it in general for all ´//@` commands adds quite a bit of complexity to the parser. I guess I could special case it just for the only/ignore parser